src/main/java/io/github/intellij/dlanguage/codeinsight/dcd/DCDCompletionClient.kt

```diff
@@ -4,6 +4,7 @@ import com.google.gson.Gson
 import com.intellij.openapi.module.ModuleUtilCore
 import com.intellij.psi.PsiFile
 import io.github.intellij.dlanguage.codeinsight.dcd.completions.DCDModel
+import org.jetbrains.rpc.LOG
```

Comment:
`org.jetbrains.rpc.LOG` isn't resolved for me. Where does that come from? The `com.intellij.openapi.diagnostic.Logger` class should be used for logging.